locale/hu.json

```diff
@@ -307,28 +307,28 @@
                 "session_time": "Szerveren töltött idő",
                 "play_time": "Játékidő",
                 "joined": "Csatlakozva",
-                "whitelisted_label": "Whitelisted",
-                "whitelisted_notyet": "not yet",
-                "btn_wl_add": "ADD WL",
-                "btn_wl_remove": "REMOVE WL",
-                "btn_wl_success": "Whitelist status changed.",
-                "log_label": "Log",
-                "log_empty": "No bans/warns found.",
-                "log_ban_count": "%{smart_count} ban |||| %{smart_count} bans",
-                "log_warn_count": "%{smart_count} warn |||| %{smart_count} warns",
-                "log_btn": "DETAILS",
-                "notes_changed": "Player note changed.",
-                "notes_placeholder": "Notes about this player..."
+                "whitelisted_label": "Engedélyezve",
+                "whitelisted_notyet": "Még nem",
+                "btn_wl_add": "Engedélyezés megadása",
+                "btn_wl_remove": "Engedélyezés visszavonása",
+                "btn_wl_success": "Az engedélyezési lista állapota megváltozott.",
+                "log_label": "Napló",
+                "log_empty": "Nem található tiltás/figyelmeztetés.",
+                "log_ban_count": "%{smart_count} kitiltás |||| %{smart_count} kitiltások",
+                "log_warn_count": "%{smart_count} figyelmeztetés |||| %{smart_count} figyelmeztetések",
+                "log_btn": "Részletek",
+                "notes_changed": "Játékos jegyzete változott.",
+                "notes_placeholder": "Jegyzetek ehhez a játékoshoz..."
             },
             "history": {
                 "title": "Kapcsolódó előzmények",
-                "btn_revoke": "REVOKE",
-                "revoked_success": "Action revoked!",
-                "banned_by": "BANNED by %{author}",
-                "warned_by": "WARNED by %{author}",
-                "revoked_by": "Revoked by %{author}.",
-                "expired_at": "Expired at %{date}.",
-                "expires_at": "Expires at %{date}."
+                "btn_revoke": "Visszavonás",
+                "revoked_success": "Művelet visszavonva!",
+                "banned_by": "Kitiltva  %{author} által",
+                "warned_by": "Figyelmeztetve %{author} által",
```

Comment:
WARNED and BANNED were all caps to improve readability in the logs page, please change back to all caps.

Reply:
All fixed